src/components/views/ProfilePage.jsx

```diff
@@ -0,0 +1,176 @@
+                    <div className="column">
+                      <div className="bio-profile">
+                        <div className="bio-profile__form">
+                          <form>
+                            <div className="field">
+                              <div className="control">
+                                <input className="input" type="text"
+                                  placeholder="First Name"
+                                  value="Sullivan" readOnly />
+                              </div>
+                            </div>
+                            <div className="field">
+                              <div className="control">
+                                <input className="input" type="text"
+                                  placeholder="Last Name" value="Ukaegbu"
+                                  readOnly />
+                              </div>
+                            </div>
+                            <div className="field">
+                              <div className="control">
+                                <input className="input" type="text"
+                                  placeholder="Username"
+                                  value="wiz surlivan" readOnly />
+                              </div>
+                            </div>
+                            <div className="field">
+                              <div className="control">
+                                <input className="input" type="text"
+                                  placeholder="Email"
+                                  value="wiztemple7@gmail.com" readOnly />
+                              </div>
+                            </div>
+                            <div className="field">
+                              <div className="control">
+                                <textarea className="textarea" readOnly>
+                                    Lorem ipsum dolor sit amet consectetur
+                                    adipisicing elit.
+                                    Expedita illo ex odio aperiam,
+                                    officiis et hic amet ad esse quae officia
+                                    culpa.
+                                    Voluptas, quaerat soluta.
+                                </textarea>
+                              </div>
+                            </div>
+                          </form>
+                        </div>
+                      </div>
+                    </div>
+                    <div className="column" />
+                    <div className="column" />
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </section>
+      </div>
+    );
+  }
+}
+
+const mapStateToProps = state => ({
+  profile: state.userProfile.profile,
+});
+
+ProfilePage.propTypes = {
+  userProfileAction: PropTypes.func.isRequired,
+  match: PropTypes.object.isRequired
```

Comment:
Remember to add `profile` in this prop-types declaration.

Reply:
This is noted. thanks